client/src/public/ts/settings.ts

```
@@ -155,6 +208,9 @@ function loadSettings(storageString: string) {
  */
 function setSettings() {
   let toSave: any = {};
```

Biome

[error] This let declares a variable that is only assigned once.

'toSave' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)